pkg/testutil/leak.go

```diff
@@ -106,6 +106,7 @@ func interestingGoroutines() (gs []string) {
 		}
 		stack := strings.TrimSpace(sl[1])
 		if stack == "" ||
+			strings.Contains(stack, "runtime/sigqueue.go") ||
```

Comment:
`created by os/signal.init` instead since that's what's launching leaky goroutine?